frontend/public/components/masthead-toolbar.jsx

```diff
@@ -369,7 +369,7 @@ class MastheadToolbar_ extends React.Component {
             <ToolbarItem className="hidden-xs">{this._renderMenu(false)}</ToolbarItem>
           </ToolbarGroup>
         </Toolbar>
-        {showAboutModal && <AboutModal isOpen={showAboutModal} closeAboutModal={this._closeAboutModal} />}
+        <AboutModal isOpen={showAboutModal} closeAboutModal={this._closeAboutModal} />
```

Comment:
Since this is no longer an `AsyncComponent`, I don't see a reason for the `showAboutModal` check here anymore. I think we can just make this change as the fix.